### DISCUSSION TITLE
Thanks for a great plugin!

### DISCUSSION THREAD

**Discussion by PatrickDanino** [iklein99/homebridge-smartthings#50] [General]
I realize you're actively working on adding features and fixing issues, but this plugin has been working very well for me and has given a second life to my numerous Smartthings devices.

Just wanted to take a moment to say thanks for all the hard work you've put into this!

**Comment by iklein99** [comment]
Thanks!Sent from my iPhoneOn Oct 22, 2022, at 4:00 PM, Patrick ***@***.***> wrote:﻿
I realize you're actively working on adding features and fixing issues, but this plugin has been working very well for me and has given a second life to my numerous Smartthings devices.
Just wanted to take a moment to say thanks for all the hard work you've put into this!

**Comment by TiganSan** [comment]
I Agree.  Thank you for writing this plugin. I am happy to buy you coffee and encourage you to continue maintaining this code.
Thank you

**Comment by mikegraben** [comment]
A little late to the party but yes, thank you @iklein99!  I sent a donation a couple nights ago - super thankful for all the work you've put into this and 'keeping the lights on' around here with Smartthings!  Thank you!
